templates/test/ci/cluster-template-prow-machine-pool-ci-version.yaml

```diff
@@ -327,9 +327,9 @@ spec:
             DEBIAN_FRONTEND=noninteractive apt-get install -y $$CI_PACKAGE=$$PACKAGE_VERSION
           done
         else
-          CI_URL="gs://k8s-release-dev/ci-periodic/$$CI_VERSION/bin/linux/amd64"
+          CI_URL="dl.k8s.io/k8s-release-dev/ci-periodic/$$CI_VERSION/bin/linux/amd64"
```

Comment:
seeing this in logs: 

```
[  113.675435] cloud-init[2044]: [2021-07-23 17:15:17] CommandException: "ls" command does not support "file://" URLs. Did you mean to use a gs:// URL?
```

It looks like we need https://github.com/spiffxp/cluster-api/commit/b2d9539802d5e325a383f611272b4abd150956e3#diff-dab9bbc5e6be4ec1c1499e40a4909de399e3f8501534b3c5b6e0a1d1a6763b40 before we can change the URLs 

@sayantani11 if you'd like, you can integrate the same changes as https://github.com/kubernetes-sigs/cluster-api/pull/4420/files#diff-dab9bbc5e6be4ec1c1499e40a4909de399e3f8501534b3c5b6e0a1d1a6763b40L34 in this PR (remove the GSUTIL install and replace the GSUTIL calls with wget) in this PR to fix it. Otherwise, I can try to make those changes in a different PR.

Reply:
okay!!! I am trying to solve this... But for clarification will the changes be same?

Reply:
They should be similar. The idea is to remove usage of GSUTIL in the ci-version yaml templates and replace it with wget.

For example, `$${GSUTIL} cp "$${CI_URL}/$${CI_PACKAGE}" "$${CI_DIR}/$${CI_PACKAGE}"` will become `wget "$${CI_URL}/$${CI_PACKAGE}" -O "$${CI_DIR}/$${CI_PACKAGE}"`.

For the `$${GSUTIL} ls` you can get rid of it and use this CI_URL instead: https://github.com/sbueringer/cluster-api/blob/b2d9539802d5e325a383f611272b4abd150956e3/test/framework/kubernetesversions/data/debian_injection_script.envsubst.sh.tpl#L94

Don't worry about changing other parts of the file that are different besides the GSUTIL changes. Let me know if you have any questions or if this doesn't make sense.